[2024-07-07T15:02:46.631+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-07T15:02:48.347+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V2.second_task scheduled__2024-07-02T00:00:00+00:00 [queued]>
[2024-07-07T15:02:49.035+0000] {local_task_job_runner.py:124} ERROR - Received SIGTERM. Terminating subprocesses
[2024-07-07T15:02:49.043+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 143
[2024-07-07T15:02:49.610+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-07T15:04:49.075+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-07T15:04:49.459+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V2.second_task scheduled__2024-07-02T00:00:00+00:00 [queued]>
[2024-07-07T15:04:49.724+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V2.second_task scheduled__2024-07-02T00:00:00+00:00 [queued]>
[2024-07-07T15:04:49.729+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-07-07T15:04:50.257+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): second_task> on 2024-07-02 00:00:00+00:00
[2024-07-07T15:04:50.340+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V2', 'second_task', 'scheduled__2024-07-02T00:00:00+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpnhk8s5et']
[2024-07-07T15:04:50.392+0000] {standard_task_runner.py:91} INFO - Job 105: Subtask second_task
[2024-07-07T15:04:50.560+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=217) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-07T15:04:50.588+0000] {standard_task_runner.py:63} INFO - Started process 236 to run task
[2024-07-07T15:04:51.349+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_V2.second_task scheduled__2024-07-02T00:00:00+00:00 [running]> on host a10fdcb539f4
[2024-07-07T15:04:53.966+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='bavenraj' AIRFLOW_CTX_DAG_ID='our_first_dag_V2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T00:00:00+00:00'
[2024-07-07T15:04:53.970+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-07T15:04:54.270+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-07T15:04:54.352+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task 2 and will be running after task1 !']
[2024-07-07T15:04:54.559+0000] {subprocess.py:86} INFO - Output:
[2024-07-07T15:04:54.669+0000] {subprocess.py:93} INFO - hey, I am task 2 and will be running after task1 !
[2024-07-07T15:04:54.745+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-07T15:04:54.803+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-07T15:04:55.722+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V2, task_id=second_task, run_id=scheduled__2024-07-02T00:00:00+00:00, execution_date=20240702T000000, start_date=20240707T150449, end_date=20240707T150455
[2024-07-07T15:04:56.265+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-07T15:04:56.510+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-07T15:04:56.555+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
